frontend/src/dialogs/MemberDialog.vue

```diff
@@ -88,13 +111,17 @@ export default {
       type: String,
       required: true
     },
-    oldName: {
+    currentName: {
```

Comment:
use `name` and `roles` as component properties and handle the change of these values internally. The internal name could be `member.name` or `memberName` or `newName`.

Reply:
renamed.